Mathlib/Analysis/LocallyConvex/WithSeminorms.lean

```diff
@@ -628,7 +627,7 @@ theorem continuous_iff_continuous_comp {q : SeminormFamily 𝕜₂ F ι'} [Topol
 #align seminorm.continuous_iff_continuous_comp Seminorm.continuous_iff_continuous_comp
 
 theorem continuous_from_bounded {p : SeminormFamily 𝕝 E ι} {q : SeminormFamily 𝕝₂ F ι'}
-    [TopologicalSpace E] (hp : WithSeminorms p) [TopologicalSpace F] (hq : WithSeminorms q)
+    {_ : TopologicalSpace E} (hp : WithSeminorms p) {_ : TopologicalSpace F} (hq : WithSeminorms q)
```

Comment:
What's the reason for this change?

Reply:
With the old version [this line](https://github.com/leanprover-community/mathlib4/blob/92a117f9897a20ea6e9175824a031349ed375b30/Mathlib/Analysis/LocallyConvex/WithSeminorms.lean#L680) doesn't work. More generally, I feel like everything goes way smoother when things that can be inferred from various places (these topologies can either be inferred from the expected type or from the types of the arguments) aren't also searched by the typeclass system, especially since Lean4 seems to be even more strict on that than Lean3.